evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -154,11 +154,11 @@ contract FluxAggregator is AggregatorInterface, Owned {
     onlyOwner()
     onlyUnenabledAddress(_oracle)
   {
-    require(oracleCount() < 42, "cannot add more than 42 oracles");
-    require(_admin != address(0), "admin address must not be 0x0");
+    require(oracleCount() < 42);
+    require(_admin != address(0));
     require(
       oracles[_oracle].admin == address(0) || oracles[_oracle].admin == _admin,
-      "cannot modify previously-set admin address"
+      ""
```

Comment:
why did we remove the error messages here?

Reply:
And if they're going to stay removed, we should be able to omit the empty string.

Reply:
Error messages put us over the contract size limit.  We're looking into better fixes in another PR, so this is hopefully temporary.  I'll remove this empty string, just an oversight.